firmware/ventilator-controller-stm32/Core/Test/Inc/Pufferfish/Test/Util.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Util.h
+ *
+ *  Created on: Nov 17, 2020
+ *      Author: Renji Panicker
+ *
+ *  A set of helper functins for writing and debugging test cases
+ */
```

Comment:
Do we need all these functions to be marked `inline` and defined in `Util.h` rather than a `Util.cpp`?